docs-web/configuration/ingress-resources/basic-configuration.md

```diff
@@ -30,10 +37,11 @@ Here is a breakdown of what this Ingress resource definition means:
 * In the `spec.tls` field we set up SSL/TLS termination:
     * In the `secretName`, we reference a secret resource by its name, `cafe‑secret`. This resource contains the SSL/TLS certificate and key and it must be deployed prior to the Ingress resource.
     * In the `hosts` field, we apply the certificate and key to our `cafe.example.com` host.
-* In the `spec.rules` field, we define a host with domain name `cafe.example.com`.
+* In the `spec.rules` field, we define a host with domain name `cafe.example.com`. The host domain name supports wildcard values for example, `foo.example.com/coffee` or `bar.example.com/tea` will be handled by the Ingress rule with  `*.example.com` host.
 * In the `paths` field, we define two path‑based rules:
-  * The rule with the path `/tea` instructs NGINX to distribute the requests with the `/tea` URI among the pods of the *tea* service, which is deployed with the name `tea‑svc` in the cluster.
-  * The rule with the path `/coffee` instructs NGINX to distribute the requests with the `/coffee` URI among the pods of the *coffee* service, which is deployed with the name `coffee‑svc` in the cluster.
+  * The rule with the path `/tea` instructs NGINX to distribute the requests with the prefix matching `/tea` URI among the pods of the *tea* service, which is deployed with the name `tea‑svc` in the cluster.
+  * The rule with the path `/tea/green` instructs NGINX to distribute the requests with the exact matching `/tea/green` URI among the pods of the *tea* service, which is deployed with the name `tea-svc` in the cluster.
+  * The rule with the path `/coffee` instructs NGINX to distribute the requests with the implementation specific matching `/coffee` URI among the pods of the *coffee* service, which is deployed with the name `coffee‑svc` in the cluster. Implementation specific matching is the default prefix matching. 
```

Comment:
Because those are new features, but we still continue to support older Kubernetes versions, let's document them in a separate section. This way people will see that the feature is only available since k8s 1.18.

What do you think about the following section? We can put it after before the Restrictions section. I used your example, but removed unnecessary `""`.

## New Features Available in Kubernetes 1.18 and Above

Starting from Kubernetes 1.18, you can use the following new features:
* The host field supports wildcard domain names, such as `*.example.com`.
* The path supports different matching rules with the new field `PathType`, which takes the following values: `Prefix` for prefix-based matching, `Exact` for exact matching and `ImplementationSpecific`, which is the default type and is the same as  `Prefix`. For example:
  ```yaml
     - path: /tea
        pathType: Prefix
        backend:
          serviceName: tea-svc
          servicePort: 80
      - path: /tea/green
        pathType: Exact
        backend:
          serviceName: tea-svc
          servicePort: 80
     - path: /coffee
        pathType: ImplementationSpecific # default
        backend:
          serviceName: coffee-svc
          servicePort: 80
   ```